azure-pipelines.yml

```diff
@@ -164,6 +164,8 @@ jobs:
     continueOnError: True
   - bash: ci/scripts/check-bazel-banned-rules.sh
     displayName: Check for banned rules
+  - bash:  ci/scripts/check_bazel_target_names.py
+    displayName: Check Bazel target names
```

Comment:
Should add:
` continueOnError: True`
here.

Reply:
Done. Also added "(Experimental)" to the checks in this area that have `continueOnError: True`.